food/ios/App/Analytics/AnalyticsManager.swift

```diff
@@ -286,6 +280,21 @@ struct AnalyticsEvent {
     return AnalyticsEvent(name: "search", category: .filter, label:mode.rawValue, cd7: cuisines, cd8: neighborhoods, cd9: bells, cd10: price, cd11: reviewer)
   }
 
+  static func appLaunched(launchOptions: LaunchOptions) -> AnalyticsEvent {
+    let direct = launchOptions?[UIApplication.LaunchOptionsKey.remoteNotification] == nil
```

Comment:
@achainan What about locally-defined triggers, like geofence events? The fix we applied to HERE accounted for those too.